tests/src/test/java/org/eclipse/hono/tests/registry/TenantAmqpIT.java

```diff
@@ -93,11 +110,28 @@ public static void shutdown(final TestContext ctx) {
     @Test
     public void testGetTenant(final TestContext ctx) {
 
-        tenantClient
-            .get(Constants.DEFAULT_TENANT)
-            .setHandler(ctx.asyncAssertSuccess(tenantObject -> {
-                ctx.assertEquals(Constants.DEFAULT_TENANT, tenantObject.getTenantId());
-            }));
+        // Prepare the identities to insert
+        final String tenantId = Constants.DEFAULT_TENANT;
+
+        final Async done = ctx.async();
+        // Insert into the device Registry
+        helper.registry.addTenant(new JsonObject()
```

Comment:
We could try to reuse TenantObject class here (and in couple more places in this class) to make this JSON Object formatting easier.